plugins/inputs/modbus/README.md

```diff
@@ -47,9 +47,16 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   # parity = "N"
   # stop_bits = 1
 
-  ## For Modbus over TCP you can choose between "TCP", "RTUoverTCP" and "ASCIIoverTCP"
-  ## default behaviour is "TCP" if the controller is TCP
-  ## For Serial you can choose between "RTU" and "ASCII"
+  ## Serial (RS485; RS232) [Windows]
+  # controller = "COM1"
+  # baud_rate = 9600
+  # data_bits = 8
+  # parity = "N"
+  # stop_bits = 1
```

Comment:
Rather than duplicate all these settings, can you add a comment above the current controller option that states for Windows do X for other OSes do Y?